pandas/tests/dtypes/test_common.py

```diff
@@ -611,8 +611,8 @@ def test__get_dtype_fails(input_param):
     ('datetime64[ns, Europe/London]', com.DatetimeTZDtypeType),
     (pd.SparseSeries([1, 2], dtype='int32'), np.int32),
     (pd.SparseSeries([1, 2], dtype='int32').dtype, np.int32),
-    (PeriodDtype(freq='D'), com.PeriodDtypeType),
-    ('period[D]', com.PeriodDtypeType),
+    (PeriodDtype(freq='D'), pd.Period),
+    ('period[D]', pd.Period),
```

Comment:
This is inevitably going to be a big PR before its through.  Might be worth separating out easy bits like this.